### PR TITLE
Update SCM for masters of CCPP + Hera Anaconda fix

### PR DESCRIPTION
**Pull request by grantfirl**
- Update GFS_typedefs.[F90/meta] to match FV3 and to work with the latest masters of ccpp-physics and ccpp-framework
 - this allows compilation of FLAKE (although it has not been tested with the SCM -- I'm not sure it can be initialized as-is)
 - change module load statements for Anaconda python for Hera after machine change

### CODE REVIEW

**Comment by grantfirl** [comment]
Address #186

File: scm/src/GFS_typedefs.F90 [hunk 1]
```diff
@@ -4104,8 +4110,13 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
        'max_lon=',max_lon,' max_lat=',max_lat,' min_lon=',min_lon,' min_lat=',min_lat,       &
        ' rhc_max=',Model%rhcmax
 
-!--- set nrcm
-    Model%nrcm = 2
+!--- set nrcm 
+
+    if (Model%ras) then
```
Comment (climbfuji) [inline scm/src/GFS_typedefs.F90:4115]: This is not new, but I assume it was forgotten when the RAS updates were brought into SCM in the first place.

File: scm/src/GFS_typedefs.F90 [hunk 2]
```diff
@@ -1493,10 +1495,6 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
 
     !--- MP quantities for 3D diagnositics 
     real (kind=kind_phys), pointer :: refl_10cm(:,:) => null()  !< instantaneous refl_10cm
-
-    !--- Extra PBL diagnostics
```
Comment (climbfuji) [inline scm/src/GFS_typedefs.F90:1497]: Is this a new change? Or just something we missed in an earlier update?

File: scm/src/GFS_typedefs.F90 (hunk 1 above)
Comment (grantfirl) [reply scm/src/GFS_typedefs.F90:4115]: Yes.

File: scm/src/GFS_typedefs.F90 (hunk 2 above)
Comment (grantfirl) [reply scm/src/GFS_typedefs.F90:1497]: I think it was missed. Just an extra variable that wasn't hurting anything.

**Pull request merged**